### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by Bigizi**
**Oop school library project**

This group is made by @Bigizi, @Habtamu210, and @alicemirigo92.
this app saves all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,33 +129,24 @@ You can deploy this project using any host of your choice
...
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-- **[How to run ruby program]**
-
-  - [use irb on command line]
+I would like to thank Microverse which provided me all knowledge about full stack development.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (danmainah) [inline README.md:152]: - [ ] @team great work on your documentation, however, the MIT license link is not working, 
image

File: Module/rental_options.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    puts 'Rental created successfully'
+  end
+
+  def list_rentals
+    id = select('Select a person from the following list by id ', 'person')
+
+    puts 'Rentals:'
+    @rentals_list.each do |rental|
+      puts "Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}" if rental.person.id == id
+    end
```
Comment (danmainah) [inline Module/rental_options.rb:42]: - [ ] @team great work on your RentalOptions class methods, however when I save a rental am unable to view it, kindly check on your output for the rentals 
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Bigizi , @Habtamu210 and @alicemirigo92 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sumairq — APPROVED** [review]
### Hi Team, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**